app/src/main/java/edu/wpi/first/shuffleboard/app/MainWindowController.java

```diff
@@ -596,6 +594,9 @@ private void newTab() {
 
   @FXML
   private void showPlugins() {
+    if (pluginStage == null) {
+      setUpPluginsStage();
```

Comment:
This was moved to here because the FXML init is now called on the launcher thread, not the application thread